addons/im_livechat/static/src/core/web/thread_patch.js

```diff
@@ -0,0 +1,45 @@
+import { Thread } from "@mail/core/common/thread";
+import { _t } from "@web/core/l10n/translation";
+import { user } from "@web/core/user";
+import { patch } from "@web/core/utils/patch";
+
+const { DateTime } = luxon;
+
+patch(Thread.prototype, {
+    get livechatVisitor() {
+        return this.props.thread.livechatVisitorMember;
+    },
+    get showVisitorDisconnected() {
+        return (
+            this.props.thread.channel_type === "livechat" &&
+            this.store.self.isInternalUser &&
+            this.props.thread.livechat_active &&
```

Comment:
Not a big deal either way, but I wonder if the condition on active is really necessary.

Reply:
I guess it is for when checking history, not being bother by the banner.